docs/_ug/commandSummary/ItemCommands.md

```diff
@@ -79,5 +82,6 @@
   | replace: ":incexample:", incexample
   | replace: ":decexample:", decexample
   | replace: ":editexample:", editexample
+  | replace: ":rmkexample:", rmkexample
   | replace: ":delexample:", delexample
 }}
```

Comment:
This file is still missing the two reassignments to `rmk` and `rmkexample` required to parse the markdown into HTML.